test/e2e/framework/framework.go

```diff
@@ -473,6 +474,38 @@ func (f *Framework) AfterEach() {
 	}
 }
 
+// DeleteNamespace can be used to delete a namespace. Additionally it can be used to
+// dump namespace information so as it can be used as an alternative of framework
+// deleting the namespace towards the end.
+func (f *Framework) DeleteNamespace(name string) {
+	defer func() {
+		err := f.ClientSet.CoreV1().Namespaces().Delete(context.TODO(), name, metav1.DeleteOptions{})
+		if err != nil && !apierrors.IsNotFound(err) {
```

Comment:
Should we log a message and simply return if `err != nil && apierrors.IsNotFound(err)` is true?

Reply:
we still need to remove it from `f.namespacesToDelete` array if `IsNotFound` is true right?